documentation/changelogs/api/2023.03.22.04.56.29.md

```diff
@@ -0,0 +1,33 @@
+# 2023.03.22.04.56.29
+
+## Improvements
+
+- Absorb `build-nginx` job into `build-images` job (#944) @dhruvkb
```

Comment:
One downside of having this as a markdown doc and not in an official release is that the reference numbers are no longer auto-linked by GitHub (i.e. `#944` won't go anywhere when this is published to our docs site). Is it possible to update the action to hard-link these?

Reply:
Good point! The easiest way is probably through this sphinx extension: https://github.com/sloria/sphinx-issues

We could also update the change template in release-drafter (https://github.com/release-drafter/release-drafter#change-template-variables) but I think the sphinx plugin would be more broadly useful in the end as it would apply to _all_ docs, not just these ones.

Reply:
Woah, what a cool extension!

Reply:
https://github.com/sloria/sphinx-issues didn't work due to it relying on a custom sphinx role, but I found this SO answer with a handy bit of code that works perfectly for us: https://stackoverflow.com/questions/72424748/myst-parser-auto-linking-linkifying-references-to-bug-tracker-issues

Tested it locally with this PR:

![image](https://user-images.githubusercontent.com/24264157/227411749-2c25a11a-b515-44cc-bd04-8aa52ad1daf4.png)

The "code" was just `Test #987` :tada: 

This is in the latest commit.